From: Easwar Hariharan <easwar.hariharan@intel.com>
Subject: staging/rdma/hfi1, IB/core: Fix LinkDownReason define for
Git-commit: e1bf0d5ecdc49cd4e2014da0d60efa74f5714fba
Patch-mainline: v4.6
References: bsc#973818, fate#319242
Acked-by: Jiri Bohac <jbohac@suse.cz>

LinkDownReason LocalMediaNotInstalled lacked an underscore
and was inconsistent with other defines in the same family.
This patch fixes this.

Reviewed-by: Ira Weiny <ira.weiny@intel.com>
Signed-off-by: Easwar Hariharan <easwar.hariharan@intel.com>
Signed-off-by: Doug Ledford <dledford@redhat.com>
---
 drivers/staging/rdma/hfi1/chip.c     |    4 ++--
 drivers/staging/rdma/hfi1/platform.c |    2 +-
 include/rdma/opa_port_info.h         |    2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

--- a/drivers/staging/rdma/hfi1/chip.c
+++ b/drivers/staging/rdma/hfi1/chip.c
@@ -5947,12 +5947,12 @@ static void handle_qsfp_int(struct hfi1_
 
 			if ((ppd->offline_disabled_reason >
 			  HFI1_ODR_MASK(
-			  OPA_LINKDOWN_REASONLOCAL_MEDIA_NOT_INSTALLED)) ||
+			  OPA_LINKDOWN_REASON_LOCAL_MEDIA_NOT_INSTALLED)) ||
 			  (ppd->offline_disabled_reason ==
 			  HFI1_ODR_MASK(OPA_LINKDOWN_REASON_NONE)))
 				ppd->offline_disabled_reason =
 				HFI1_ODR_MASK(
-				OPA_LINKDOWN_REASONLOCAL_MEDIA_NOT_INSTALLED);
+				OPA_LINKDOWN_REASON_LOCAL_MEDIA_NOT_INSTALLED);
 
 			if (ppd->host_link_state == HLS_DN_POLL) {
 				/*
--- a/drivers/staging/rdma/hfi1/platform.c
+++ b/drivers/staging/rdma/hfi1/platform.c
@@ -816,7 +816,7 @@ void tune_serdes(struct hfi1_pportdata *
 		} else
 			ppd->offline_disabled_reason =
 			   HFI1_ODR_MASK(
-				OPA_LINKDOWN_REASONLOCAL_MEDIA_NOT_INSTALLED);
+				OPA_LINKDOWN_REASON_LOCAL_MEDIA_NOT_INSTALLED);
 		break;
 	default:
 		dd_dev_info(ppd->dd, "%s: Unknown port type\n", __func__);
--- a/include/rdma/opa_port_info.h
+++ b/include/rdma/opa_port_info.h
@@ -97,7 +97,7 @@
 #define OPA_LINKDOWN_REASON_WIDTH_POLICY			41
 /* 42-48 reserved */
 #define OPA_LINKDOWN_REASON_DISCONNECTED			49
-#define OPA_LINKDOWN_REASONLOCAL_MEDIA_NOT_INSTALLED		50
+#define OPA_LINKDOWN_REASON_LOCAL_MEDIA_NOT_INSTALLED		50
 #define OPA_LINKDOWN_REASON_NOT_INSTALLED			51
 #define OPA_LINKDOWN_REASON_CHASSIS_CONFIG			52
 /* 53 reserved */
